static/test/javascripts-legacy/helpers/svg-paths.js

```diff
@@ -0,0 +1,45 @@
+define(function() {
```

Comment:
Agreed. There's only 15 usages, looks very trivial. I'll add a task to the inbox